code/modules/awaymissions/gateway.dm

```diff
@@ -150,6 +158,15 @@ obj/machinery/gateway/centerstation/process()
 	var/ready = 0
 	var/obj/machinery/gateway/centeraway/stationgate = null
 
+/obj/machinery/gateway/centeraway/New()
+	..()
+
+	gateways.Add(src)
+
+/obj/machinery/gateway/centeraway/Destroy()
+	..()
+
+	gateways.Remove(src)
```

Comment:
This should be before the ..(), Destroy inheritance should always go in reverse.